components/brave_wallet/browser/json_rpc_response_parser.cc

```diff
@@ -70,4 +72,11 @@ bool ParseBoolResult(const std::string& json, bool* value) {
   return false;
 }
 
+void ConvertSingleUint64Result(std::string* raw_response) {
```

Comment:
I think this should return a string, out params should generally be avoided

Reply:
also at the very least we should convert the potentially empty string to absl::optional in the wrapper here

Reply:
I removed this function from this PR, will add this conversion after rebase on master in another PR